src/frontend/js/components/ReferralActivityIndicator/ActivityUrgencyLevelChanged.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { defineMessages, FormattedMessage, FormattedDate } from 'react-intl';
+
+import { GenericErrorMessage } from 'components/GenericErrorMessage';
+import { Spinner } from 'components/Spinner';
+import { useReferralUrgencies } from 'data';
+import { getUserFullname } from 'utils/user';
+import { ReferralActivity, ReferralUrgencyLevelHistory } from 'types';
+
+const messages = defineMessages({
+  loadingUrgencies: {
+    defaultMessage: 'Loading urgency options...',
+    description:
+      'Accessible text for the spinner while loading urgency options in the referral form',
+    id: 'components.ReferralForm.UrgencyField.loadingUrgencies',
+  },
+  urgencyLevelChanged: {
+    defaultMessage:
+      'The expected response date move from { oldUrgencyLevel } to { newUrgencyLevel }',
+    description: 'Urgency level changed.',
+    id: 'components.ReferralActivityIndicator.ActivityUrgencyLevelChanged',
+  },
+  urgencyLevelChangedExplanation: {
+    defaultMessage: 'Explanation: { explanation } ',
+    description: ' explnanation of Urgency level changment.',
+    id:
+      'components.ReferralActivityIndicator.ActivityUrgencyLevelChanged.urgencyLevelChangedExplanation',
+  },
+});
+
+interface ActivityUrgencyLevelChangedProps {
+  referralurgencylevelhistory: ReferralUrgencyLevelHistory;
+}
+
+export const ActivityUrgencyLevelChanged: React.FC<ActivityUrgencyLevelChangedProps> = ({
+  referralurgencylevelhistory,
+}) => {
+  const {
+    data: Referralurgencies,
+    status: statusUseReferralUrgency,
+  } = useReferralUrgencies();
+  switch (statusUseReferralUrgency) {
+    case 'error':
+      return <GenericErrorMessage />;
+
+    case 'idle':
+    case 'loading':
+      return (
+        <Spinner size="small">
+          <FormattedMessage {...messages.loadingUrgencies} />
+        </Spinner>
+      );
+    case 'success':
+
+    case 'success': {
+      return (
+        <>
+          <span className="block">
+            <FormattedMessage
+              {...messages.urgencyLevelChanged}
+              values={{
+                oldUrgencyLevel: Referralurgencies?.results.find(
```

Comment:
Rather than showing the urgency levels themselves, we could try yo be more helpful for our users and show the expected dates.

There's a dependency, named `luxon` which should provide functions to calculate the old & new expected dates.